db/migrate/20221025072655_add_email_strategy_to_subdomain.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailStrategyToSubdomain < ActiveRecord::Migration[6.1]
+  def change
+    add_column :subdomains, :email_strategy, :boolean, default: false
```

Comment:
@Sushmit1997 this should not be a boolean. Its very unclear what it does. 

Please change it to an enum, and name it email_notification_strategy-- enum values should be: 'system-email' , 'user-email'. Add a validation to prevent anything but 'system-email' , 'user-email'

Reply:
I've changed it to enum. Please re-review.